src/components/horizonCard/HorizonCard.ts

```diff
@@ -42,6 +42,10 @@ export class HorizonCard extends LitElement {
     this.processLastHass()
   }
 
+  public getCardSize (): number {
+		return 6
```

Comment:
Hey, thanks for the PR! This feels hard to maintain though. Without any reference or comment to contextualize the number we will surely forget to update it once a style change would require it. Do you see a way to calculate "6" from css properties? Additionally I'd suggest to add a link to the relevant HA docu as docstring to the function.

Reply:
The size isn't fixed as it can change based on what fields and whether a title has been configured. It should either remain as is (not implementing the method) or take into account the configuration when returning a value. Since I'm already working on a major redesign I suggest we leave the current version as is and address this in the upcoming major release.

Reply:
> Without any reference or comment to contextualize the number we will surely forget to update it once a style change would require it.

I've added nice docs, thanks

> Additionally I'd suggest to add a link to the relevant HA docu as docstring to the function.

This should be `@see` i guess? I don't really use jsdoc that much

> The size isn't fixed as it can change based on what fields and whether a title has been configured.

This is now dynamic, thanks